project/Dependencies.scala

```diff
@@ -11,7 +11,7 @@ object Dependencies {
   val ScalaVersions = Dependencies.Scala2Versions :+ Dependencies.Scala3
   val AkkaVersion = System.getProperty("override.akka.version", "2.9.2")
   val AkkaVersionInDocs = AkkaVersion.take(3)
-  val AkkaPersistenceJdbcVersion = "5.2.0" // only in migration tool tests
+  val AkkaPersistenceJdbcVersion = "5.4.0" // only in migration tool tests
```

Comment:
does this mean that we can run tests for migration with Scala 3 now? Then this can be changed in nightly.yml:
```
- { scalaVersion: "3.3",  jdkVersion: "1.17.0", jvmName: "temurin:1.17.0", extraOpts: '', testCmd: "core/test"}
```

Reply:
Changed in [30ad14c](https://github.com/akka/akka-persistence-r2dbc/pull/541/commits/30ad14c1c877d96162ae545bed471dc7b552050e)